analyses/molecular-subtyping-CRANIO/01-craniopharyngiomas-molecular-subtype.Rmd

```diff
@@ -61,10 +61,16 @@ output_subtype <- file.path(results_dir, "CRANIO_molecular_subtype.tsv")
 
 ```{r load-inputs-please-wait}
 # Inputs
+## Read pathology diagnosis json file
+path_dx_list <- jsonlite::fromJSON(
+  file.path(root_dir, "analyses", "molecular-subtyping-CRANIO", "cranio-subset",
+            "CRANIO_subtyping_path_dx_strings.json")
+)
+
 ## Read in metadata
 metadata <-
   read_tsv(file.path(root_dir, "data", "histologies-base.tsv")) %>%
-  filter(pathology_diagnosis == "Craniopharyngioma")
+  filter(pathology_diagnosis %in% path_dx_list$exact_path_dx)
```

Comment:
I'm not sure if anything will go wrong here, but since this is only a list of one, we might keep the `==` here. I have seen instances of using `%in%` with one thing and it not work properly. Seems this works as expected - no change in results, but just want to be more full-proof, and if we add more dx later, we can update?